README.md

```diff
@@ -1 +1,44 @@
 # Governance for Pyro, NumPyro, and related projects
+
+Pyro is owned by the [LF AI Foundation](https://wiki.lfaidata.foundation/display/PYRO/Pyro+Home).
```

Comment:
minor nit: github link in "Reference Information"  section still points to the old uber url.